source/pages/Home/Panel/ActivityPanel.tsx

```diff
@@ -5,14 +5,27 @@ import { useStore } from 'hooks/useStore';
 import { PanelList } from './components/PanelList';
 
 export const ActivityPanel: FC = () => {
-  const { activeAccount } = useStore();
+  const { activeAccount, activeNetwork, accounts } = useStore();
 
   return (
     <>
       <div className="p-4 w-full h-full text-white text-base bg-bkg-3">
-        {activeAccount &&
-        activeAccount.transactions &&
-        Object.values(activeAccount.transactions).length > 0 ? (
+        {activeNetwork.chainId === 1 || activeNetwork.chainId === 4 ? (
```

Comment:
here we should check if the network chain is ethereum, like

const isSyscoinChain = Boolean(networks.syscoin[activeNetwork.chainId]);

if !isSyscoinchain then we can call the function to get the history for account according to the current chain, the request for assets and txs must work for all of custom rpcs/evm networks